web/api_isvcs.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2016 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package web
+
+import (
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/control-center/serviced/dao"
+	"github.com/control-center/serviced/domain"
+	"github.com/control-center/serviced/domain/service"
+	"github.com/control-center/serviced/health"
+	"github.com/control-center/serviced/isvcs"
+
+	"github.com/zenoss/go-json-rest"
+)
+
+func getAllInternalServices(w *rest.ResponseWriter, r *rest.Request, ctx *requestContext) {
+	results := []interface{}{}
+
+	for _, service := range getISVCS() {
+		if service.ID == isvcs.InternalServicesISVC.ID {
+			results = append(results, getParent())
+		} else {
+			results = append(results, getService(service))
+		}
+	}
+
+	w.WriteJson(results)
```

Comment:
`web.writeJSON` is used inconsistently throughout the `web` package, but maybe that should be used here instead of directly calling the `ResponseWriter`'s method.